mlx_hw_steering_parser.py

```diff
@@ -118,9 +119,10 @@ def parse_args():
     if (args.dump_hw_resources):
         print("-hw is not supported yet.")
         sys.exit(0)
-    if (args.dpdk_pid != -1):
-        print("-pid is not supported yet.")
-        sys.exit(0)
+    if (args.dpdk_pid > 0):
+        if dr_trigger.trigger_dump(args.dpdk_pid, args.dpdk_port, args.FILEPATH, 0) is None:
```

Comment:
lets check after trigger, when we open the dump file, this way we will cover both cases, triggered and non-triggered